src/components/tasks/card/index.tsx

```diff
@@ -514,62 +514,57 @@ const Card: FC<CardProps> = ({
                     )}
                 </div>
             </div>
-            {showAssignButton() ? (
-                <AssigneeButton />
-            ) : (
-                <div className={classNames.contributor}>
-                    <span className={classNames.cardSpecialFont}>
-                        Assigned to
-                    </span>
-                    <span className={classNames.contributorImage}>
-                        <Image
-                            src={assigneeProfileImageURL}
-                            alt={cardDetails.assignee || DUMMY_NAME}
-                            width={30}
-                            height={30}
-                        />
-                    </span>
-                    {isEditable ? (
-                        isUserAuthorized && (
-                            <div className={classNames.suggestionDiv}>
-                                <input
-                                    data-testid="assignee-input"
-                                    ref={inputRef}
-                                    value={assigneeName}
-                                    className={classNames.cardStrongFont}
-                                    onKeyDown={(e) => {
-                                        handleChange(e, 'assignee');
-                                    }}
-                                    onChange={(e) => {
-                                        handleAssignment(e);
-                                        debounce(
-                                            fetchUsers,
-                                            400
-                                        )(e.target.value);
-                                    }}
-                                    role="button"
-                                    tabIndex={0}
-                                />
-
-                                {isLoadingSuggestions ? (
-                                    <Loader />
-                                ) : (
-                                    showSuggestion && (
-                                        <SuggestionBox
-                                            suggestions={suggestions}
-                                            onSelectAssignee={handleClick}
-                                        />
-                                    )
-                                )}
-                            </div>
-                        )
-                    ) : (
-                        <span className={classNames.cardStrongFont}>
-                            {cardDetails.assignee}
-                        </span>
-                    )}
-                </div>
-            )}
+
+            <div className={classNames.contributor}>
+                <span className={classNames.cardSpecialFont}>
```

Comment:
Do not use span, this could be a `<p>` tag. 
Use semantic tags as long as possible.

Reply:
done